collector/boot_time_bsd.go

```diff
@@ -28,11 +28,11 @@ type bootTimeCollector struct {
 }
 
 func init() {
-	registerCollector("boottime", defaultEnabled, newBootTimeCollector)
+	registerCollector("boottime", defaultEnabled, newBootTimeCollector(logger))
```

Comment:
This logger seems suspect, not sure where its coming from or how it would get updated.